satosa/README-short.txt

```diff
@@ -0,0 +1 @@
+SATOSA is a configurable proxy for translating between different authentication protocols such as SAML2, OpenID Connect, and OAuth2.
```

Comment:
Unfortunately Hub requires this to be no more than 100 characters :sweat_smile:

Reply:
Thanks!  Done in 2c0950a.